.github/workflows/publish_release.yml

```diff
@@ -14,78 +15,69 @@ jobs:
   packaging:
     name: Build wheels on ${{ matrix.os }}
     runs-on: ${{ matrix.os }}
-    if: startsWith(github.ref, 'refs/tags/') || (github.event_name == 'pull_request' && github.event.pull_request.merged == true)
+    if: startsWith(github.ref, 'refs/tags/') || (github.event_name == 'pull_request' && github.event.pull_request.merged == true) || github.event_name == 'workflow_dispatch'
     strategy:
       matrix:
-        os: [ubuntu-20.04, windows-2019, macos-10.15]
+        os:
+          - ubuntu-latest
+          # - windows-latest
```

Comment:
Please remove commented code, otherwise it might be forgotten and will live on forever.
If you need it again, you can always get it from the git history.

Reply:
Done.